managers/tests/test_unit_model.py

```diff
@@ -0,0 +1,48 @@
+from django.test import TestCase
+from managers.commons.constants import *
+from managers.models import Project
+from django.core.exceptions import ValidationError
+from django.utils import timezone
+
+
+class TestProjectModel(TestCase):
+    def test_that_project_should_have_start_date(self):
+        project = Project(
+            name='X',
+        )
+        with self.assertRaisesRegexp(
+            ValidationError,
+            "This field cannot be null.",
+        ):
+            project.full_clean()
+
+    def test_that_project_name_should_not_be_blank(self):
+        project = Project(
+            start_date=timezone.now(),
+        )
+        with self.assertRaisesRegexp(
+                ValidationError,
+                "This field cannot be blank.",
+        ):
+            project.full_clean()
+
+    def test_that_project_name_should_not_be_longer_than_MAX_NAME_LENGTH(self):
+        project = Project(
+            start_date=timezone.now(),
+            name='X'*(MAX_NAME_LENGTH+1),
+        )
+        with self.assertRaisesRegexp(
+            ValidationError,
+            "Ensure this value has at most " + str(MAX_NAME_LENGTH) + " characters",
+        ):
+            project.full_clean()
+
+    def test_that_project_name_should_be_longer_than_MAX_NAME_LENGTH(self):
```

Comment:
All letters should be lowercase

Reply:
What do you test here? Name should **not** be longer than `MAX_NAME_LENGTH`.
And test is not failing because validation error will not be risen.
So this test checks if creating `Project` instance actually work?

Reply:
Done